server/fleet/activities.go

```diff
@@ -677,7 +684,27 @@ func (a ActivityTypeMDMUnenrolled) Documentation() (activity string, details str
 }`
 }
 
-// AuthzType implement AuthzTyper to be able to verify access to activities
-func (*Activity) AuthzType() string {
-	return "activity"
+type ActivityTypeEditedMacOSMinVersion struct {
+	TeamID         *uint   `json:"team_id"`
+	TeamName       *string `json:"team_name"`
+	MinimumVersion string  `json:"minimum_version"`
+	Deadline       string  `json:"deadline"`
+}
+
+func (a ActivityTypeEditedMacOSMinVersion) ActivityName() string {
+	return "edited_macos_min_version"
+}
+
+func (a ActivityTypeEditedMacOSMinVersion) Documentation() (activity string, details string, detailsExample string) {
+	return `Generated when the minimum required macOS version is modified.`,
```

Comment:
should we mention that the activity will also be created if the deadline is modified?

Reply:
I'm merging to unblock Gabe tomorrow, but feel free to put in another PR if you want to change the language here.